### ISSUE TITLE
x/sys/windows: TestProcThreadAttributeHandleList failing with "The parameter is incorrect." on release-branch.go1.16

### ISSUE THREAD

**Issue by bcmills** [golang/go#50160]
```
--- FAIL: TestProcThreadAttributeHandleList (0.00s)
    syscall_windows_test.go:1047: The parameter is incorrect.
FAIL
FAIL	golang.org/x/sys/windows	1.629s
```

`greplogs --dashboard -md -l -e 'FAIL: TestProcThreadAttributeHandleList' --since=2021-12-01`

[2021-12-13T22:30:07-03aa0b5-f1f3923/windows-386-2008](https://build.golang.org/log/4a7f27b60c348a931890e54616c3f707d5d3d510)
[2021-12-13T22:30:07-03aa0b5-f1f3923/windows-amd64-2016](https://build.golang.org/log/30734bbba3181af41dbcf640f2245093c0a1275f)
[2021-12-13T22:30:07-03aa0b5-f1f3923/windows-amd64-race](https://build.golang.org/log/c18eca0fb651205e4959a513c1275123d6d4f9c0)

**Comment by bcmills** [comment]
This is a release-blocker for the next Go 1.16 minor release via #11811.
The failing test was added in [CL 371276](https://golang.org/cl/371276).

It can be addressed by either fixing `x/sys` to work with Go 1.16, or backporting fixes to Go 1.16 to fix the failing test.

CC @zx2c4 @bufflig @golang/release.

**Comment by zx2c4** [comment]
Fixed with https://golang.org/cl/371694. Sorry about that.
